cmd/peer-rest-server.go

```
@@ -1048,6 +1048,59 @@ func (s *peerRESTServer) ReloadPoolMetaHandler(w http.ResponseWriter, r *http.Re
 	}
 }
 
+func (s *peerRESTServer) StopRebalanceHandler(w http.ResponseWriter, r *http.Request) {
+	if !s.IsValid(w, r) {
+		s.writeErrorResponse(w, errors.New("invalid request"))
+		return
+	}
+
+	objAPI := newObjectLayerFn()
+	if objAPI == nil {
+		s.writeErrorResponse(w, errServerNotInitialized)
+		return
+	}
+	pools, ok := objAPI.(*erasureServerPools)
+	if !ok {
+		s.writeErrorResponse(w, errors.New("not a multiple pools setup"))
+		return
+	}
+
+	pools.StopRebalance()
```

We should return an error here ?